python/lsst/cp/pipe/cpCombine.py

```diff
@@ -349,13 +363,26 @@ def run(self, inputExpHandles, inputScales=None, inputDims=None):
 
         self.combine(combinedExp, inputExpHandles, expScales, stats)
 
-        # The calibration should _never_ have NO_DATA set.
+        # The calibration should _never_ have NO_DATA set, as this is
+        # handled poorly downstream, and so we raise here after
+        # explicitly checking for pixels with that bit set.
         if self.config.checkNoData:
             test = ((combinedExp.mask.array & afwImage.Mask.getPlaneBitMask("NO_DATA")) > 0)
             if (nnodata := test.sum()) > 0:
-                raise RuntimeError(f"Combined calibration has {nnodata} pixels!")
+                raise RuntimeError(f"Combined calibration has {nnodata} NO_DATA pixels!")
+
+        if self.config.censorMaskPlanes:
+            # Any mask planes that are not the noGoodPixelsMask plane
+            # should be cleared.  This should remove things like the
+            # CROSSTALK plane from printing into the calibration.  Run
+            # after the checkNoData pass so that we don't censor what
+            # we want to raise on.
+            mask = combinedExp.mask
+            for plane in mask.getMaskPlaneDict().keys():
+                if plane != self.config.noGoodPixelsMask:
+                    mask.clearMaskPlane(plane)
 
-        self.interpolateNans(combined)
+        self.interpolateNans(combined, self.noGoodPixelsMask)
```

Comment:
Can you explicitly use the keyword here, `maskPlane=self.noGoodPixelsMask`?

Reply:
Yes, and I've fixed the missing `.config`.